client/galaxy/scripts/layout/scratchbook.js

```diff
@@ -171,7 +171,7 @@ return Backbone.View.extend({
             window.open( options.url );
         } else if ( options.target == '_top' || options.target == '_parent' || options.target == '_self' ) {
             window.location = options.url;
-        } else if ( !this.active || options.noscratchbook ) {
+        } else if ( !this.active || options.noscratch ) {
```

Comment:
This change looks incorrect, menu.js is still using 'noscratchbook'.  (perhaps an unintentional change?)